### PR TITLE
fix: watch extra item width, padding and line to show for useMaxVisibleSections hook

### PR DESCRIPTION
**Pull request by ychhabra-eightfold**
## SUMMARY:
fix: watch extra item width, padding and line to show for useMaxVisibleSections hook

## CHANGE TYPE:

- [ ] Bugfix Pull Request
- [ ] Feature Pull Request

## TEST COVERAGE:

- [ ] Tests for this change already exist
- [ ] I have added unittests for this change

### CODE REVIEW

File: src/hooks/useMaxVisibleSections.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { useEffect, useState } from 'react';
+import React, { useEffect, useState } from 'react';
```
Comment (mfazil-eightfold) [inline src/hooks/useMaxVisibleSections.ts:1]: why `React` import is needed?

Comment (ychhabra-eightfold) [reply src/hooks/useMaxVisibleSections.ts:1]: React import should be there on every react component

Comment (mfazil-eightfold) [reply src/hooks/useMaxVisibleSections.ts:1]: The hook does not require React import, only needed if you use any JSX.

Comment (ychhabra-eightfold) [reply src/hooks/useMaxVisibleSections.ts:1]: line no 4,5 have React.MutableRefObject, so it needs the react import

**Review by mfazil-eightfold — APPROVED** [review]

**Pull request merged**